### PR TITLE
write test  "should be possible to select on columns inside a through ta...

### PR DESCRIPTION
**Pull request by ErikvdVen**

### CODE REVIEW

File: test/include/findAll.test.js [hunk 1]
```diff
@@ -1750,6 +1750,32 @@ describe(Support.getTestDialectTeaser('Include'), function() {
...
+      var self = this;
+      this.fixtureA(function() {
+        self.models.Product.findAll({
+          attributes: ['title'],
+          include: [
+            {
+              model: self.models.Tag,
+              through: {
+                where: {
+                  ProductId: 3
```
Comment (mickhansen) [inline test/include/findAll.test.js]: Bit of a contrived example but i suppose it proves your point ;)

**Comment by mickhansen** [comment]
Thank you @ErikvdVen, it demonstrates the case so that is very helpfull :)

**Comment by mickhansen** [comment]
heh, we didn't actually have `through.where` support at all, but now we do.

**Comment by ErikvdVen** [comment]
Great to hear! We will test it asap ;) Thanks!

**Pull request merged**